app/src/main/res/values/strings.xml

```diff
@@ -468,7 +468,7 @@
   <!-- TextViewBindingAdapters -->
   <string name="just_now">just now</string>
   <string name="last_logged_in_recently">recently</string>
-  <string name="time_ago">%s ago</string>
```

Comment:
This string is no longer needed, please delete it.

Reply:
done